dependencies/vaticle/repositories.bzl

```diff
@@ -20,6 +20,6 @@ load("@bazel_tools//tools/build_defs/repo:git.bzl", "git_repository")
 def vaticle_dependencies():
```

Comment:
Update to dependencies that gives us access to the token-based `deploy_npm` rule